### PR TITLE
feat(prettier): allow to override & extends config

### PR DESCRIPTION
**Pull request by frosato-ekino**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,20 +12,40 @@
...
 
-## How to use
+## How to use scripts
 
 ```json
 "scripts": {
     "format": "knuckle prettier 'src/**/*.{js,json}' --write",
     "lint": "knuckle eslint src",
     "lint:fix": "knuckle eslint src --fix"
+```
```
Comment (GuillaumeAmat) [inline README.md]: It misses a closing bracket.

Comment (GuillaumeAmat) [reply README.md]: It breaks the markdown

Comment (GuillaumeAmat) [reply README.md]: Apart from that, it's ok for me. Good job!

Comment (frosato-ekino) [reply README.md]: it's fixed now :)

**Pull request merged**